doc/python_reference.rst

```diff
@@ -771,6 +771,7 @@ Source Space Data
 
    head_to_mni
    compute_morph_matrix
+   compute_morph_sdr
```

Comment:
This does not link properly

https://8332-1301584-gh.circle-artifacts.com/0/html/python_reference.html

You need to add `from .source_estimate import compute_morph_sdr` in `mne/__init__.py` as these are all top-level functions here. Also in your example you can then use `mne.compute_morph_sdr` directly.

Reply:
check